students/KevinWaldron/lesson04/mailroom.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+
+# donor data structure, a dictionary of donors, each donor name is key and the value is a list constaining the total donations
+# (for easy sorting), followed by the history of donations
+donors = {
+"Bill Ted":[726.63, 353.53, 348.1, 25.00],
+"Frank Fred":[178.76, 120.50, 56.76, 1.50],
+"Laura Todd":[5.75, 5.75],
+"Steve Lincoln":[165.28, 75.38, 89.9],
+"Lisa Grant":[209.7, 175.50, 34.20]
+}
+
+def list_donors():
+    """ Prints a list of the donors by name """
+    print("\n" + ("-"*10) + " Donors " + ("-"*10))
+    for donor in donors:
+        print(donor)
+
+def quit_menu():
+    """ Handles the menu quit selection """
+    return "Quit"
+
+def create_thank_you(donation):
+    """
+    Prints a thank for you the given donor and amount
+    :param donation: dictionary containing name of the donor to thank and the amount donated
+    :returns: formatted thank you string
+    """
+    return f"""Dear {donation["name"]},\nThank you for your very generous donation of ${donation["amount"]:.2f}.  It \nwill go very far in supporting the Human Fund, \"Money for \nPeople.\"\n{"Sincerely":>40}\n{"Art Vandelay":>50}"""
+
+def donor_entry():
+    """ Handles donor entry menu selection """
+    name = input("\nEnter donor's full name or 'q' to return to the previous menu: ")
+    if(name == "q"):
+        return
+    amount = input("Enter donation amount or 'q' to return to the main menu: ")
+    if (amount == "q"):
+        return
+    if(not name in donors):
```

Comment:
more pythonic -> `if name not in donors`

also no need for parens in if statements.